readme.md

```diff
@@ -1,6 +1,6 @@
 ## Very Very Simple Youtube Downloader.
 
-just a chrome extension that downloads youtube videos from https://9xbuddy.org/
+just a chrome extension that downloads youtube videos from [9xbuddy](https://9xbuddy.org/)
 
```

Comment:
I just like to have that as a non formated link (for reference to the link, so don't change that)